chart/tests/carto.yaml

```diff
@@ -0,0 +1,56 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "https://github.com/CartoDB/carto3-helm/archive/refs/tags/v0.0.4.tar.gz",
+          "path": "/chart"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "linter-packaging",
+          "params": {
+            "kind": "HELM"
+          }
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "https://github.com/CartoDB/carto3-helm/releases/download/v0.0.4/carto3-helm.tar.gz",
```

Comment:
I don't know how we are going to integrate these tests yet, but if the release is hardcoded, how do we plan to manage it? updating all these files on each release?

Reply:
Ok let me clarify, this `tar.gz` is an intermediate artifact between different phases with dependencies built-in. We just used the "release attachment" feature of GitHub to upload the `tar.gz` somewhere that can be accessed by the verification service during the `verify` phase.

Currently we are working to pass this tarball between phases, once it is done this value should not be needed.

Reply:
I think there's a misunderstanding here. `verify` phase' context is made of the following main components at the moment:

* `resources`: where the tests that will be executed are located at. This should point at a URL with the tests intended for the application being verified.
* `application`: the definition of the app to be verified.
* `runtime_parameters`: base64 encoded values to customize the app deployment for verification (current `application.values`)

Once `helm-package` is available, you'll be able to remove `application` and let the pipeline verify the result of the new action. However `resources` should be kept.

Reply:
Thank you @paleloser I didn't have such detailed information about this feature, now it sounds crystal clear to me. I will include your proposed changes to the PR.

Reply:
Done!